script/Verify_11155111_to_4801_part2.ts

```diff
@@ -0,0 +1,55 @@
+import dotenv from "dotenv";
+import hre from "hardhat";
+
+dotenv.config();
+
+const aggregatorsFactory = process.env.L1_SEPOLIA_AGGREGATORS_FACTORY || "";
+const l2Target = process.env.OP_MESSAGES_INBOX || "";
+const crossDomainMsgSender =
+  process.env.OPTIMISM_SEPOLIA_CROSS_DOMAIN_MESSENGER || "";
```

Comment:
This should be `WORLD_CHAIN_SEPOLIA_CROSS_DOMAIN_MESSENGER` instead of `OPTIMISM`